charts/k8s-monitoring/charts/feature-application-observability/README.md

```diff
@@ -40,6 +40,41 @@ Be sure perform actual integration testing in a live environment in the main [k8
 
 ## Values
 
+### Connectors: Grafana Cloud Host Info
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| connectors.grafanaCloudMetrics.enabled | bool | `true` | Generate host info metrics from telemetry data. These metrics are required for using Application Observability in Grafana Cloud. Note: Enabling this may incur additional costs. See https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/pricing/ |
```

Comment:
This wording look good to you, @rlankfo ?

Reply:
looks good to me! 🎸